src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4841indentation/ClassWithChainedMethods.java

```diff
@@ -15,16 +15,20 @@ public ClassWithChainedMethods getInstance(String... uselessParams) { //indent:2
 
   public static void main(String[] args) { //indent:2 exp:2
     new ClassWithChainedMethods().getInstance("string_one") //indent:4 exp:4
-    .doNothing("string_one".trim(), //indent:4 exp:8 warn
+    // violation below ''method call' child has incorrect indentation level 4, expected .* 8.'
+    .doNothing("string_one".trim(), //indent:4 exp:8
                "string_two"); //indent:15 exp:>=8
 
     int length = new ClassWithChainedMethods("param1", //indent:4 exp:4
                                 "param2").getInstance() //indent:32 exp:>=8
-    .doNothing("nothing") //indent:4 exp:>=8 warn
-    .length(); //indent:4 exp:>=8 warn
+    // violation below ''.' has incorrect indentation level 4, expected level should be 8.'
+    .doNothing("nothing") //indent:4 exp:>=8
+    .length(); //indent:4 exp:>=8
```

Comment:
Please remove all trailing comments like this that has violation comment , it was our first version of bdd.
Now in message we have all this information about expected and detected

Reply:
done https://github.com/checkstyle/checkstyle/blob/175b96311d77b7782b63e867898725668e7c9c99/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4841indentation/ClassWithChainedMethods.java#L24-L27